### PR TITLE
Optional setting for maximum time spent on a single puzzle, add Least pv to filter.py

### PR DESCRIPTION
**Pull request by NelloHo**

### CODE REVIEW

File: filter.py [hunk 1]
```diff
@@ -14,7 +14,10 @@ def line_count(filename):
 
 def filter(annotations, min, max, values):
     for k, v in min.items():
-        if float(annotations.get(k, 0)) < float(v):
+        if k == 'pv':
+            if len(annotations.get(k, 0).split(',')) < int(v):
```
Comment (ianfab) [inline filter.py:18]: The default value of get looks wrong here, considering that the value should be a string.

**Review by ianfab — COMMENTED** [review]
Thanks for your PR. I just added one small comment.

File: filter.py (hunk 1 above)
Comment (NelloHo) [reply filter.py]: I'm sorry about the bug. Fixed.

**Comment by ianfab** [comment]
Thanks

**Pull request merged**